azurestack/test_utils.go

```diff
@@ -0,0 +1,22 @@
+package azurestack
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func getArmResourceNameAndGroup(s *terraform.State, name string) (string, string, error) {
+	rs, ok := s.RootModule().Resources[name]
+	if !ok {
+		return "", "", fmt.Errorf("Not found: %s", name)
+	}
+
+	armName := rs.Primary.Attributes["name"]
```

Comment:
It might be best to also check here for hasName? as well as that name is not empty